index.html

```diff
@@ -16,6 +16,7 @@
 	<script src="https://unpkg.com/react@16/umd/react.development.js" crossorigin></script>
 	<script src="https://unpkg.com/react-dom@16/umd/react-dom.development.js" crossorigin></script>
 	<script src="https://unpkg.com/babel-standalone@6/babel.min.js"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.20.1/moment.min.js"></script>
```

Comment:
Nice. The more dependencies we can add for this, the better.


Feels even better when it's obsolete/useless: https://momentjs.com/docs/#/-project-status/